services/stackexchange/stackexchange.tester.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const ServiceTester = require('../service-tester')
+const { isMetric } = require('../test-validators')
+
+const t = new ServiceTester({ id: 'stackexchange', title: 'StackExchange' })
+
+module.exports = t
+
+t.create('Invalid t parameter')
+  .get('/stackoverflow/t/invalidimage.json')
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'stackoverflow',
+      value: 'invalid',
+    })
```

Comment:
When everything is literal, these can be written a little more simply (as you've done with the networkOff test):

```js
  .expectJSON({ name: 'stackoverflow', value: 'invalid' })
```

Reply:
Resolved with newest commit!